pkg/swagger/swagger.go

```diff
@@ -126,6 +148,19 @@ func Run(api, outputDir string) error {
 		}
 
 		s.Definitions[azureResource].Properties = properties
+
+		if g.Features().systemData {
```

Comment:
this doesn't look right.  I think it is masking the fact that we actually need to add this to the model and deal with it.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

Reply:
And https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html

Reply:
Added separate PR for this. Sill in draft, but working on it https://github.com/Azure/ARO-RP/pull/1412